bin/Phantom/submit_phantom.py

```diff
@@ -811,19 +811,63 @@ def gridpackGeneration (debugging):
     # launch the submission script
     execute ('source ' + submitfilename, debugging)
 
-    # wait until all jobs finish, before calculating the cross-section
-    # and compressing the gridpack
-    finished = False
-    while not finished:
-        finished = True
-        unfinished = int (0)
-        for fil in processoutputs:
-            if not os.path.exists (fil):
-                finished = False
-                unfinished += 1
-        sys.stdout.write ('waiting for: ' + str (unfinished) + ' jobs\r' )
-        sys.stdout.flush ()
-        time.sleep (60) # seconds
+    os.chdir (rootfolder)
+    sys.exit (0)
+
+
+# ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ----
+
+''' creating tarball of the gridpack generated for the wmLHE prodcution framework
+'''
+def gridpackCreation (debugging):
+    rootfolder = os.getcwd ()
+    # values to be inserted in the phantom r.in file
+    substitute = {}
+    config = ConfigParser.ConfigParser ()
+    config.optionxform = str # to preserve the case when reading options
+    print 'reading config file:',sys.argv[1]
+    config.read (sys.argv[1])
+
+    # prepare the folder for the gridpack creation
+    foldername = os.getcwd () + '/phantomGrid_' + datetime.datetime.now().strftime('%y-%m-%d-%H-%M')
```

Comment:
Hello @govoni 

Thanks for reviewing. 
May be I'm miss understanding your comments, please make me clear if I'm wrong.

According to my understanding :
1) do you mean we should keep the older version of script to run quickly in one go for validation and also keep the changes I have made?
2) Or do you mean I revisit my changes and try to make it more independent on human intervention like to check whether all the jobs finished?

About the prepare folder for the gridpack creation line, I have forgot to remove it. I can remove that part from the script in next iteration.

Reply:
Hello @skeshri ,

I think we should include an option in the gridpackGeneration which calls also gridpackCreation when jobs are finished, before quitting the script, if required. 
Option 2 seems more complicated to me, since you would like to have a kind of daemon than runs even when you are disconnected, and checks whether jobs are finished. You could try to send the case of option 1 on LSF, though.